### PR TITLE
intial integration redis-rs

### PR DESCRIPTION
**Pull request by commented-line**
Redis-rs has 5 million downloads 12k downloads per day. Redis-rs is the most popular redis library for rust. It is also used as dependecy for many rust libraries.

### CODE REVIEW

**Comment by commented-line** [comment]
@Navidem @jonathanmetzman need approval for build tests and merge

**Comment by Navidem** [comment]
Waiting for the panel to approve the merge.

File: projects/redis-rs/Dockerfile
Comment (jonathanmetzman) [inline projects/redis-rs/Dockerfile:18]: There's no point in this line.

Comment (code-terror) [reply projects/redis-rs/Dockerfile]: Removed the line..

**Review by Navidem — APPROVED** [review]
LGTM

**Pull request merged**
